src/Microsoft.AspNet.Mvc.TagHelpers/MissingAttributeLoggerStructure.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Mvc.TagHelpers
+{
+	internal class MissingAttributeLoggerStructure : ILoggerStructure
+	{
+		private readonly string _uniqueId;
+	    private readonly IEnumerable<string> _missingAttributes;
+	    
+	    public MissingAttributeLoggerStructure(string uniqueId, IEnumerable<string> missingAttributes)
+	    {
+	        _uniqueId = uniqueId;
+	        _missingAttributes = missingAttributes;
+	    }
+	    
+	    string ILoggerStructure.Message
```

Comment:
I can't even remember why I did this now but I thought I had a reason. Put it down to not having VS to tell me otherwise at the time :)